indexing-service/src/test/java/io/druid/indexing/overlord/TaskLifecycleTest.java

```diff
@@ -109,8 +125,25 @@
 import java.util.concurrent.Executor;
 import java.util.concurrent.TimeUnit;
 
+@RunWith(Parameterized.class)
 public class TaskLifecycleTest
 {
+
+  @Parameterized.Parameters(name = "taskStorageType={0}")
+  public static Collection<String[]> constructFeed()
+  {
+    return Arrays.asList(new String[][]{{"HeapMemoryTaskStorage"}, {"MetadataTaskStorage"}});
```

Comment:
I agree with @nishantmonu51 that these should be classes instead of Strings as we do for the other parameterized tests.